src/libslic3r/PrintConfig.cpp

```diff
@@ -918,6 +919,7 @@ void PrintConfigDef::init_fff_params()
     def->category = L("Quality");
     def->tooltip = L("Decrease this value slightly(for example 0.9) to reduce the amount of material for bridge, "
                      "to improve sag");
+    def->sidetext = L("x"); // ORCA add side text
```

Comment:
I find add "x" side text strange.
Can we remove x side texts?

Reply:
there is a % used for percentage. x fits well for ratio imo. also i tried to show values as "1.00" to indicate its float numbers but they always rounded to "1"

Reply:
`%` and other units like `mm` are common and therefore look natural.
The x looks very strange and confusing as it displays like `1     x`.
We shouldn't add it.

Reply:
google suggest few alternatives for multiplier symbol.
Multiplication sign definition: the symbol ( ⋅ ), ( × ), or ( ∗ )
But ratio is completely different thing. it mostly suggest proportional ratio and its symbol is " : "

```
1     mm
1     x
1     ⋅
1     ×
1     ∗
```

ofc i would not die if we dont add it. still looks nice imo :)

here is degree, mm, % and x in same view

![Screenshot-20240525033427](https://github.com/SoftFever/OrcaSlicer/assets/28517890/24839e12-ff77-4221-93df-87bd6e58faff)

Reply:
I still think the x is confusing here :)